interfaces/cython/cantera/examples/onedim/diffusion_flame_batch.py

```diff
@@ -27,10 +28,17 @@ class FlameExtinguished(Exception):
     pass
 
 
-# Create directory for output data files
-data_directory = 'diffusion_flame_batch_data/'
-if not os.path.exists(data_directory):
-    os.makedirs(data_directory)
+hdf_output = find_spec('h5py') is not None
```

Comment:
For a function that's only going to be used once, I think it's more clear to just write `import importlib` above and use the fully qualified function name here.